firmware/network/border_router_node/include/border_router_node.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @{
+ * @ingroup     network
+ * @file        border_router_node.h
+ * @brief       this module content all functions of Border Router node
+ * @author      RocioRojas <rociorojas391@gmail.com>
+ *
+ */
+
+#ifndef BORDER_ROUTER_NODE_H
+#define BORDER_ROUTER_NODE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * @brief   This is a dummy function, because the important part of this module is
+ *          Makefile.dep
+ */
+int dummy_function (void);
+
+#ifdef __cplusplus
+}
+#endif
+#endif /* BORDER_ROUTER_NODE_H */
+       /**@}*/
```

Comment:
why do you need this file?

Reply:
Yes, if you read the doc.txt, the important thing is the Makefile.dep